e2e/helper/product_helper.js

```diff
@@ -0,0 +1,29 @@
+
+const searchProduct = async (page, productName) => {
+    await page.click('#search')
+    await page.type('#search', productName)
+    await page.locator('#search').press('Enter');
+
+}
+
+const selectProduct = async page => {
+    await page.click('#product-item-info_1')
```

Comment:
Por conta do Magento gerar um id dinâmico de acordo com o product_id, essa regra atual funcionará apenas caso o product_id for igual a 1. Creio que para o fluxo funcionar melhor deveria buscar pelo primeiro item da lista:
`await page.click('.product-item-info')`